plugins/base/src/main/kotlin/signatures/KotlinSignatureUtils.kt

```diff
@@ -0,0 +1,24 @@
+package org.jetbrains.dokka.base.signatures
+
+import javaslang.Tuple2
+import org.jetbrains.dokka.base.translators.documentables.PageContentBuilder
+import org.jetbrains.dokka.model.Documentable
+import org.jetbrains.dokka.model.ExtraModifiers
+import org.jetbrains.dokka.model.ExtraModifiers.Companion.kotlinOnlyModifiers
+import org.jetbrains.dokka.model.properties.WithExtraProperties
+
+object KotlinSignatureUtils : JvmSingatureUtils {
```

Comment:
Looks inconsistent, `KotlinSignatureUtils` should also be an interface

Reply:
I need it to be instance to be part of delegation mix-in